packages/liveblocks-react-lexical/src/liveblocks-plugin-provider.tsx

```diff
@@ -70,10 +70,12 @@ export function useEditorStatus(): EditorStatus {
 
   const subscribe = useCallback(
     (onStoreChange: () => void) => {
-      if (provider === undefined) return () => {};
+      if (provider === undefined) return () => { };
+      provider.on("status", onStoreChange);
       provider.on("sync", onStoreChange);
       return () => {
         provider.off("sync", onStoreChange);
+        provider.off("status", onStoreChange);
       };
```

Comment:
Is it okay to remove the subscription to `provider.on("sync", onStorageChange)` now that `provider.on("status", onStorageChange)` does the same thing and more?

Reply:
yea, probably